main.py

```diff
@@ -312,6 +312,10 @@ def print_message(message, message_type=None):
     email = os.environ['INPUT_COMMITTER_EMAIL']
     # Actions that should not be updated
     ignore = os.environ['INPUT_IGNORE']
+    # Commit message
+    commit_message = os.environ['INPUT_COMMIT_MESSAGE']
+    # Pull Request Title
+    pr_title = os.environ['INPUT_PULL_REQUEST_TITLE']
```

Comment:
I think we should pass these to the `GitHubActionsVersionUpdater` class `__init__()` method, just like `ignore` input.

Reply:
I am very much a newbie with Python, I will look into this and figure it out :)

Reply:
Hi @saadmk11 - added it but I am not sure it was in the spirit of your suggestion.  Let me know and thank you!